app/views/reportings/new.html.erb

```diff
@@ -37,7 +37,8 @@ See doc/COPYRIGHT.rdoc for more details.
   <%= javascript_include_tag "timelines_select_boxes.js" %>
 <% end %>
 
-<h2><%= l('timelines.new_reporting').tap { |t| html_title t } %></h2>
```

Comment:
WTF!

Thanks for replacing that thing...
Evidence for the fact that shorter is not always better ^^